docs/guide/search.md

```diff
@@ -473,9 +579,10 @@ ez_search_engine_solr:
                 default: endpoint0
 ```
 
-##### Shared Core example
+##### Shared core example
```

Comment:
Naming seems to me a little bit inconsistent. Few lines above we have 'Single-core' and here we have `Shared core`. I'd use 'Single Core' and 'Shared Core' respectively. Core is a Solr term, hence the capital letter.

Reply:
As for capitalization, Solr doesn't capitalize "core" in its own doc, e.g. https://lucene.apache.org/solr/guide/6_6/solr-cores-and-solr-xml.html

But fixed hyphenation (it should be hyphenated, because it serves as an adjective here)

Reply:
Ok, you won :+1: :grinning: